.github/workflows/release.yml

```diff
@@ -210,14 +182,6 @@ jobs:
       - name: Load cheqd-node image
         run: docker load -i cheqd-node-image.tar
 
-      - name: Download testnet image
-        uses: actions/download-artifact@v2
-        with:
-          name: cheqd-testnet-image.tar
-
-      - name: Load testnet image
-        run: docker load -i cheqd-testnet-image.tar
-
       - name: Login to docker
```

Comment:
Use the Github Actions recommended way using the specific actions and secret.GITHUB_TOKEN instead, as this is safer than using a PAT: https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages

Reply:
Yeah... Wanted to do it too. Finally managed to configure package permissions.